series/series_test.go

```diff
@@ -381,6 +454,18 @@ func TestSeries_Compare(t *testing.T) {
 			[]bool{false, false, false},
 			Bools([]bool{false, false, true}),
 		},
+		{
+			Times([]time.Time{time.Now(), time.Now().Add(time.Hour * 240)}),
+			In,
+			"01/25/2017",
+			Bools([]bool{false, false}),
+		},
+		{
+			Times([]time.Time{time.Now(), time.Now().Add(time.Hour * -240)}),
+			In,
+			[]string{"25/01/2017", "01/31/2017"},
```

Comment:
This is not the right time format to be parsed!

From the docs:

`RFC3339     = "2006-01-02T15:04:05Z07:00"`